packages/adblocker-benchmarks/blockers/adblockpluscore-index.js

```diff
@@ -0,0 +1,9 @@
+const { contentTypes } = require('./contentTypes.js');
```

Comment:
The default `index.js` does not export all of these objects. It has a different API.